sample/src/main/assets/users-config.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
This json also should have apiKey. Currently it's provided in `build.gradle`, but should be moved here to keep all users information in a single place. So structure might be like this:
```json
{
  apps: [
    {
      apiKey:"",
      users: [{id:""}, {id:""}]
    }
  ]
}
```